setup.py

```diff
@@ -4,7 +4,7 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-VERSION = os.environ.get("RAY_PROVIDER_VERSION", "0.2.0")
```

Comment:
@richardliaw I missed this on the last one. I don't think we should force this into the environment. I'd say if we want to make a global version, doing something like this would be cleaner 
https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package